### ISSUE TITLE
Ender 3 v2 4.2.2 Mainboard - Broken Layout after updating to 2.0.9

### ISSUE THREAD

**Issue by PfefferT** [MarlinFirmware/Configurations#601]
Hello,

I have tried to update my display with the current firmware, but after that the icons are missing, the layout is yellow and text is cut (see screenshot). I used the DWIN_SET folder from ender 3 v2 example configuration.

Did I use the wrong folder?

![PXL_20211028_215458057](https://user-images.githubusercontent.com/13468090/139380696-449c5462-a4bd-4024-8fc5-af854fabd694.jpg)

**Comment by ellensp** [comment]
When updating the LCD firmware the sd card needs to be formatted with fat32 and 4k block sizes. Or it will not update the lcd correctly . The 4k block size is a requirement.

**Comment by ellensp** [comment]
when updating the lcd, it should go blue for a while then Orange.  If it goes Red something wen't wrong.

**Comment by PfefferT** [comment]
![grafik](https://user-images.githubusercontent.com/13468090/139383152-986cb0af-6868-4b11-9504-b512d1c6f9d5.png)

![grafik](https://user-images.githubusercontent.com/13468090/139383202-1b26f228-ed37-4625-bb59-c0ed1609c535.png)

In my case the display goes orange not red. But perhaps making it green would make it more determinable xD

**Comment by ellensp** [comment]
That is parts of bootloader on the lcd and cannot be changed easily.

**Comment by PfefferT** [comment]
ok good to know.

as you can see above the filesystem is fat 32 and blocksize is 4096. is it possible that the sd card itself is not compatible? I am using the sd card which was delivered with the printer.

**Comment by PfefferT** [comment]
So I bought today a new Micro SD card from Intenso with 16GB. I formatted it slowly on Windows to FAT32 and 4096 blocksize. I copied then the DWIN_SET Folder from Creality Ender 3 V2 examples folder from Configuration 2.0.9.2 to the Micro SD card. I have plugged in the card into the display and powered on the printer. the display went from blue to orange. I powered off. After unpluggin and restarting the printer I got the same issue on the display like in my screenshot above.

so what is causing this? Did I do something wrong?

**Comment by thinkyhead** [comment]
Last I checked that `DWIN_SET` and Marlin 2.0.9.x were still happy together whether using Creality UI, Enhanced UI, Jyers UI, or Marlin UI. Maybe it's the firmware that needs proper updating now that you have the `DWIN_SET` installed. Try all the different UI variants to see whether they are all broken, or if some work.

**Comment by tome9111991** [comment]
It look you use a old printer firmware.

You only update display?

**Comment by fgietzen** [comment]
I just experienced the same issue after upgrading from 2.0.7. The screen went orange and shows that I am using firmware 2.0.9.2, so I guess flashing worked fine?  
*Note: I used branch "2.0.9.2" in both repos*

Marlin UI seem to have no errors but seems to be less fancy and rotated by 90°. I guess this is normal?

After going to 2.0.8 with Creality UI (using the same formatted sd card), everything looks normal now.
